### ISSUE TITLE
How to enable HTTPS?

### ISSUE THREAD

**Issue by Synchro** [scaleway-community/scaleway-gitlab#32]
I've been trying to see if there's some way to enable HTTPS automatically with letsencrypt on the scaleway-gitlab image. I can see in the gitlab.rb config file that there is provision for doing exactly this for mattermost, but not for gitlab itself. I know I *could* install certbot, configure nginx to answer web verifications, place keys and certs in the right places (e.g. setting `nginx['ssl_certificate']`), and configure auto-renewal etc, but this seems to be entirely against the point of having an automated setup like this, plus figuring out how to make changes that won't get broken by updates in an automated build is considerably more difficult than a typical static setup. Pointers?

**Comment by Synchro** [comment]
I installed [acme.sh](https://github.com/Neilpang/acme.sh) and requested a certificate successfully (using DNS auth so I wouldn't need to mess with nginx config). I set all the relevant HTTPS config options I could find in `gitlab.rb` and ran `gitlab-cal reconfigure` and... nothing. Not the slightest sign of any HTTPS-related config appeared in the newly-generated nginx config. How is this supposed to work?
